en/stops.md

```diff
@@ -5,8 +5,8 @@
 | stop_id | Stops that are in different physical locations (i.e., different designated precise locations for vehicles on designated routes to stop, potentially distinguished by signs, shelters, or other such public information, located on different street corners or representing different boarding facility such as a platform or bus bay, even if nearby each other) should have different `stop_id`.
 | | `stop_id` is an internal ID, not intended to be shown to passengers. |
 | | Maintain consistent `stop_id` for the same stops across data iterations (see [Dataset Publishing & General Practices](#dataset-publishing--general-practices)). |
-| stop_name | The `stop_name` should match the agency's public name for the stop, station, or boarding facility, e.g. what is printed on a timetable, published online, and/or presented at the location.  |
-| | When there is not a published stop name, follow consistent stop naming conventions throughout the feed.  |
+| stop_name | The `stop_name` should match the agency's public name for the stop (or platform), station, entrance/exit, or boarding area (e.g., what is printed on a timetable, published online, and/or presented at the location).  |
+| | When there is not a published stop name, follow consistent stop naming conventions throughout the feed:<ul><li>Stops named after cross streets should use "at", "&" or "/" to indicate cross streets (e.g., "Main at 8th", "Main & 8th", "Main / 8th"). </li><li>Stops named after addresses should be in simplified form (e.g., "231 Route 89").</li><li>Stops named after landmarks should avoid using commercial names that may not be permanent. Civic names are suggested (e.g., "Library").</li><li>Long narrative names are discouraged (e.g., "Sacramento St across from S Pine St at Tour of Nevada City Bicycle Shop"). If necessary, narrative names should be succinct and meaningful.</li></ul>   |
```

Comment:
Can we consolidate to one convention for the group `['at','&','/']`? 

My preference would be: `<main facility> at <cross-street>` 

Where the cross-street may have different names on either side: `<main facility> at <cross-street 1> / <cross-street 2>`

Reply:
I agree that `at` is the best single convention if we're going to settle on one, primarily because it's the easiest to handle with text-to-speech (TTS) engines. In [OneBusAway Alexa](https://github.com/OneBusAway/onebusaway-alexa) I had to write specific code to convert all these various options (I've also seen `+` and `@`) into something that made sense when read out loud. This applies to virtual assistants like Alexa and Google Assistants, but also accessibility engines like Android TalkBack and Apple VoiceOver. It's also one less thing for agencies to change when they start supplying the `tts_stop_name` field specifically for TTS engines as well (which could include other tweaks for abbreviations, etc.).

Reply:
I updated the PR to consolidate the Best Practice to recommend only "at".

@e-lo Using "/" to indicate different names of a cross street on either side of the main street may cause confusion since "/" is sometimes used to indicate cross streets. It may produce ambiguous instances like `main facility / cross street 1 / cross street 2`.